symplyphysics/laws/astronomy/change_in_apparent_magnitude_from_distance.py

```diff
@@ -0,0 +1,51 @@
+from sympy import (Eq, solve, log, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The apparent magnitude is a measure of the brightness of a celestial body (more precisely, the illumination created
+## by this body) from the observer's point of view. The brighter the object, the smaller its magnitude.
+## The relationship of the stellar magnitude school with real physical quantities is logarithmic, since a change in brightness
+## by the same number of times is perceived by the eye as a change by the same amount.
+## The difference in the stellar magnitudes of two objects is equal to the decimal logarithm of the ratio of their illuminances,
+## up to a multiplier.
+
+## Law is: m2 - m1 = -2.5 * lg(L2 / L1), where
```

Comment:
Is there any reason not to remove minus and write it as `m1 - m2`?

Reply:
> Is there any reason not to remove minus and write it as `m1 - m2`?

This is a more generally accepted form of writing, judging by the sources. And also m2 is a larger quantity than m1, in a physical sense. Because the smaller the magnitude, the brighter the object. So it turns out that from the point of view of brightness, less is subtracted from more.